cnames_active.js

```diff
@@ -3312,6 +3312,7 @@ var cnames_active = {
   "woodsman": "haximilian.github.io/woodsman",
   "wooyun": "jiji262.github.io/wooyun_articles",
   "woq": "nicamoq.github.io/WoqHentai",
+  "word": "worddictionary.vercel.app",
```

Comment:
```suggestion
  "word": "cname.vercel-dns.com", // noCF
```
As the CI error indicates, for sites on Vercel you'll want to target `cname.vercel-dns.com` with the Cloudflare proxy disabled